public/app/features/library-panels/state/api.ts

```diff
@@ -90,9 +90,10 @@ export async function addLibraryPanel(
 
 export async function updateLibraryPanel(panelSaveModel: PanelModelWithLibraryPanel): Promise<LibraryElementDTO> {
   const { libraryPanel, ...model } = panelSaveModel;
-  const { uid, name, version } = libraryPanel;
+  const { uid, name, version, folderId } = libraryPanel;
   const kind = LibraryElementKind.Panel;
   const { result } = await getBackendSrv().patch(`/api/library-elements/${uid}`, {
+    folderId,
```

Comment:
🤔 we have [this other PR](https://github.com/grafana/grafana/pull/58393/files#diff-f3bc123a92bfa4aa9371dc4b06800bff62598140f08a8c5538f3c3bff627f7abR13) who is deprecating the usage of `folderId` and is starting to use `folderUid`, is it possible to use that here? or should we wait until that PR is merged?

Reply:
Might be worth waiting until that PR is merged